checkov/main.py

```diff
@@ -15,6 +15,7 @@
 from configargparse import Namespace
 from urllib3.exceptions import MaxRetryError
 
+import checkov.logging_init
```

Comment:
```suggestion
import checkov.logging_init  # should be imported before the others to ensure correct logging setup
```
please add some extra info, so the next developer doesn't move it 😄